packages/neon-js/src/experimental/helpers.ts

```diff
@@ -224,6 +224,18 @@ export async function addFees(
     );
   }
 
+  if (config.networkFeeOverride && config.prioritisationFee) {
```

Comment:
can we do the precondition checks at the start of the method so we can do an early exit.